filetests/isa/x86/extend-i128-run.clif

```diff
@@ -0,0 +1,26 @@
+test run
+target x86_64
+
+function u0:0() -> b1 system_v {
```

Comment:
nit: here and below, can you remove `system_v` in the signature? otherwise it might trigger an error on systems using other ISAs.

Reply:
```suggestion
function u0:0() -> b1 {
```